Src/coffeescript/cql-execution/src/elm/comparison.coffee

```diff
@@ -1,5 +1,6 @@
 { Expression } = require './expression'
 { Uncertainty } = require '../datatypes/datatypes'
+{ IncompatibleTypesException } = require './quantity'
 
 # Equal is completely handled by overloaded#Equal
```

Comment:
Should we also have similar logic in `Equal` (e.g., should `85 'mmol/L' = 65 '%'` return null)?

Reply:
Good catch. I'll go implement that too.